sources/packages/backend/apps/api/src/route-controllers/student/student.students.controller.ts

```diff
@@ -195,6 +195,11 @@ export class StudentStudentsController extends BaseController {
     description:
       "Requested file was not found or the user does not have access to it.",
   })
+  @ApiForbiddenResponse({
+    description:
+      "Warning: This file has not been scanned and will be available to download once it is determined to be safe or " +
+      "Error: The original file was deleted due to security rules. Please re-check file and attempt to upload again.",
```

Comment:
It can be adjusted as below:
"the original file was deleted due to security rules."

These messages are for swagger documentation and they do not need to be a copy/paste from the original messages. Either way, it is not a blocker.